src/python/pants/backend/python/subsystems/pytest.py

```diff
@@ -89,6 +89,13 @@ def register_options(cls, register):
                 "to tests under this environment variable name."
             ),
         )
+        register(
+            "--config",
+            type=file_option,
+            default=None,
+            advanced=True,
+            help="Path to pytest.ini or alternative Pytest config file",
```

Comment:
It'd be helpful to include the restrictions you described in the PR description, something like this maybe:

> Pytest will attempt to auto-discover the config file, meaning that it should typically be an ancestor of your tests, such as in the build root. Pants will not automatically set `--rootdir` for you to force Pytest to pick up your config file, but you can manually set `--rootdir` in `[pytest].args`. Refer to https://docs.pytest.org/en/stable/customize.html#initialization-determining-rootdir-and-configfile.

When adding this, you'd want to use `\n\n` in between the first sentence and the next sentence. Use soft wrapping otherwise. For example:

https://github.com/pantsbuild/pants/blob/d44d43ce116e8851c63b3b84cbcda0712d9b473e/src/python/pants/option/global_options.py#L822-L825

Reply:
Bump on this, as it's definitely a gotcha.

Reply:
Fixed in latest commit!